ecommerce-ios/Extensions/Color+Application.swift

```diff
@@ -34,6 +34,9 @@ extension Color {
     /// #EFEFEF
     static var galleryGray: Color { Color(hex: 0xEFEFEF) }
 
+    /// #F9FBFD Gray 100
+    static var gray100: Color { Color(hex: 0xF9FBFD) }
```

Comment:
Please use this page to name the color 😂 https://chir.ag/projects/name-that-color/#F9FBFD